### ISSUE TITLE
Deleting (some) realms return 502

### ISSUE THREAD

**Issue by mpanik** [GoogleCloudPlatform/healthcare-federated-access-services#35]
Try:
1. Use realm `ddap-ic-admin_journey_WorkflowJourneyTes`
2. Send DELETE request
3. Observe `502`

Note that we are able to remove new realms, but not the existing one from above (the realm is too big?).

**Comment by cdvoisin** [comment]
Possible timeout or something as we delete 400 or 500 entries at a time in a loop. Likely this should be a long-running background operation. Can you confirm it only happens on realms with lots of data, such as many user accounts, login states, etc?

**Comment by mpanik** [comment]
@cdvoisin yes I do confirm. If the deletion is done in background maybe DAM/IC could return right away `202` and a `Location` header where we can fetch the status of deletion. wdyt?

**Comment by cdvoisin** [comment]
We'd likely need to build some additional infrastructure for this, likely leveraging our current "process" background stuff like "keyGC" uses. Perhaps this is out of scope for this phase but let's revisit with the next bundle of work. Does that work for you?
